portfolio/src/main/webapp/script.js

```diff
@@ -35,18 +35,31 @@ function newCommentBox(comment){
   const commentBox = document.createElement('div');
   const commentName = document.createElement('p');
   const commentText = document.createElement('p');
+  const commentReport = document.createElement('a');
 
-  // set text for elements
+  // set info for elements
   commentName.innerText = comment.name + ':';
   commentText.innerText = comment.text;
+  commentReport.innerText = "Report";
+  commentReport.onclick = function() {
+    addReport(comment.id);
```

Comment:
Nit: In the back-end you are adding reports but in the front-end to the users they are reporting a comment. I recommend renaming to reportComment or something along those lines.